### PR TITLE
docs: added integration example doc for App Header, to aid asoption.

### PR DESCRIPTION
**Pull request by dstack**
Reviewer: @aarmour

### CODE REVIEW

File: demos/basic.html [hunk 1]
```diff
@@ -32,6 +32,6 @@
 </div>
 
 <script src="/bundles/js?modules=o-app-header:/demos/src/demo.js"></script>
-<script src="https://origami.pearsoned.com/registry/embedapi?autoload=resize"></script>
+<script src="//registry.origami.ft.com/embedapi?autoload=resize"></script>
```
Comment (aarmour) [inline demos/basic.html]: Undo this change for all of the demos.

**Comment by dstack** [comment]
@aarmour ack, demo changes removed

File: examples/INTEGRATION.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
+# Integrating o-app-header
+
+This document is designed to help you get o-app-header working in your application.
+
+### Core Concept
+
+The idea is to initialize App Header in a way that allows it to work for users, even if you are not aware of user session state, or application state, and then change things as you become aware of those states.  The App Header is an active widget;  it's meant to be changed and upgraded alongside the user experience of your app.  The techniques that follow work off of this core concept, and are effective regardless of when you initialize piSession.ja and/or the AppHeader.
```
Comment (aarmour) [inline examples/INTEGRATION.md]: typo: piSession.ja

File: examples/INTEGRATION.md [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+
+// this assignment adds the instance to the DOM, non-globally, so that it can later be retrieved and manipulated.
+// header.o-app-header will exist after O.DOMContentLoaded has fired.
+document.querySelector('header.o-app-header').oAppHeader = appHeader;
+```
+
+Once you have a stored reference to the oAppHeader instance, retrieving it is trivial.
+
+```js
+var appHeader = document.querySelector('header.o-app-header').oAppHeader;
```
Comment (aarmour) [inline examples/INTEGRATION.md]: I wouldn't recommend adding properties to DOM elements as a general practice. The likelihood of collisions is low, but it feels dirty...

File: examples/INTEGRATION.md [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+appHeader.setMode('Basic', {
+  user: {
+    givenName: 'UserFirstname'
+  }
+});
+```
+
+**Note:** setMode() may be called at any time once oAppHeader is initialized.  This is useful for switching the display of the header to respond to auth events, and changes to user profile data in the event they change their given name in the application.
+
+O-app-header is completely de-coupled from piSession.js, and expects data to be passed into it for correct rendering.  This was done to ensure that developers can integrate the header without having to first use a specific integration of piSession.js.  Auth, session management, and user-profile handling are left entirely to the integrating application, and the data is simply passed to the header as options.
```
Comment (aarmour) [inline examples/INTEGRATION.md]: typo: big `O`
